bin/u-boot-test-console

```diff
@@ -32,4 +32,9 @@ if [ -z "${console_baud}" ]; then
     console_baud=115200
 fi
 
+# Force a reset into FEL mode
+if [ "${recovery_impl}" == "sunxi-fel" ]; then
```

Comment:
Hmm. Not sure what I think about this. It's adding type-specific logic into a generic file. Also, since this runs picocom after the echo, it risks (a) not capturing any output generated from processing the reset command (b) not having configured the serial device for the correct baud rate when sending "reset".

BTW, what is the echo command communicating with; a console mux daemon, the Sunxi device's boot ROM, ...?